packages/elements/src/constants.ts

```diff
@@ -49,6 +49,7 @@ export const HttpSecuritySchemeColors: Dictionary<string, HttpSecurityScheme['ty
   http: 'orange',
   oauth2: 'red',
   openIdConnect: 'blue',
+  mutualTLS: 'grey',
```

Comment:
grey seems fine, no? :laughing:

Reply:
Grey is always fine, except that [in Tailwind it is `gray`](https://tailwindcss.com/docs/text-color) (unless we have it overridden)

We use `blue` for it in `v7` though, so you may just pick that.

Reply:
`blue` is occupied by `openIdConnect`, thus perhaps `yellow` provided that it's readable enough?